<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Javascript Test Environment</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>15</number>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Document:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="inspectorButton">
            <property name="text">
             <string>Web Inspector</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWebView" name="documentView">
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0,1,0,0,1">
          <item>
           <widget class="QPushButton" name="run1Button">
            <property name="text">
             <string>Run 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="commandLine1"/>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="run2Button">
            <property name="text">
             <string>Run 2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="commandLine2"/>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="run3Button">
            <property name="text">
             <string>Run 3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="commandLine3"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="javascriptTabs">
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Script"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Docx"/>
    <addaction name="actionOpen_Javascript"/>
    <addaction name="actionSave_Current"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpen_Docx">
   <property name="text">
    <string>Open Docx</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionOpen_Javascript">
   <property name="text">
    <string>Open Script</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_Current">
   <property name="text">
    <string>Save Current</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNew_Script">
   <property name="text">
    <string>New Script</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
